front/src/applications/stdcm/components/StdcmResults/SimulationReportSheet.tsx

```diff
@@ -24,8 +25,8 @@ const getStopType = (step: StdcmPathStep, t: TFunction) => {
   return capitalizeFirstLetter(t(`stdcm:trainPath.stopType.${step.stopType}`));
 };
 
-const getArrivalTime = (step: StdcmPathStep, t: TFunction) => {
-  if (!step.isVia) {
+export const getArrivalTimes = (step: StdcmPathStep, t: TFunction, shouldDisplay: boolean) => {
```

Comment:
Why do you export it ?
```suggestion
const getArrivalTimes = (step: StdcmPathStep, t: TFunction, shouldDisplay: boolean) => {
```